pkg/knownguids/guids.go

```diff
@@ -432,11 +465,13 @@ var GUIDs = map[guid.GUID]string{
 	*guid.MustParse("243D3E8C-2780-4A25-9693-A410475BFCEC"): "FrameBufferBltLib",
 	*guid.MustParse("2A40F516-C852-4BAA-B7A8-0E9EA090D659"): "FrameBufferBltLib",
 	*guid.MustParse("B2F0D71A-A39F-4094-854B-0C6BA6910CCE"): "FrameworkUefiLib",
+	*guid.MustParse("DF0FCD70-264A-40BF-BBD4-06C76DB19CB1"): "Fsp22SecCoreS",
 	*guid.MustParse("98C8588C-640A-4BB4-AEA0-3F81CDE17524"): "FspDxeIpl",
 	*guid.MustParse("BC59E2E1-7492-4031-806E-C48DCCC3A026"): "FspInitPeim",
+	*guid.MustParse("890B12B4-56CC-453E-B062-4597FC6D3D8C"): "FspMeasurementLib",
 	*guid.MustParse("8714C537-6D4B-4247-AA6C-29E8495F9100"): "FspNotifyDxe",
 	*guid.MustParse("29CBB005-C972-49F3-960F-292E2202CECD"): "FspNotifyPhasePeim",
-	*guid.MustParse("1BA0062E-C779-4582-8566-336AE8F78F09"): "FspSecCore/ResetVector/XenResetVector/ResetVector/ResetVector/ResetVector/SecCore",
+	*guid.MustParse("1BA0062E-C779-4582-8566-336AE8F78F09"): "FspSecCore/ResetVector/XenResetVector/ResetVector/ResetVector/ResetVector/SecCore/ResetVector/XenResetVector/ResetVector/ResetVector/ResetVector/SecCore",
```

Comment:
It looks like the import tool could be improved a bit. The `/` means there are multiple names for the GUID. The problem is the import tool shouldn't list ResetVector multiple times.